### DISCUSSION TITLE
Bypass for OpenVino AI Plugins for MacOS

### DISCUSSION THREAD

**Discussion by dimako88** [intel/openvino-plugins-ai-audacity#272] [Q&A]
So I've tried to follow instructions of brave @eukarpov on installing plugins directly to MacOS Audacity ... but failed (

Currently I am using CrossOver, made a bottle of Windows Audacity and installed the latest plugin [v3.6.1-R3.1](https://github.com/intel/openvino-plugins-ai-audacity/releases/tag/v3.6.1-R3.1). If I choose to install any models - I got an error after a long download

So I ended up with option "no models installed" and then just manually copy models from Models folders of the older release [[v3.4.2-R1](https://github.com/intel/openvino-plugins-ai-audacity/releases/tag/v3.4.2-R1)] And it worked, though with the base model only and running on the limited virtual hardware of CrossOver. 

May I PLEASE ask someone to upload same  Models folders of the LARGE-V3 model with all xml, bin files?

Maybe Mac user community will also benefit from this method

Thank You!

**Comment by dimako88** [comment]
Found it by myself and would be happy to share:
https://huggingface.co/Intel/whisper.cpp-openvino-models/tree/main

**Comment by RyanMetcalfeInt8** [comment]
Hey, just a tip about installing models manually. If you look at our Inno Setup script (which is what is used to generate the installer), you can see the URL to all of the models (xml, bin, etc.). Specifically, starting here: https://github.com/intel/openvino-plugins-ai-audacity/blob/main/tools/windows/audacity_ai_plugins.iss#L219
